File: problemas/iniciante/1180/1180.cs
using System;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine().Trim());
        string[] X = Console.ReadLine().Trim().Split(' ');

        int menor_valor = int.Parse(X[0]);
        int posicao = 0;
        for(int i = 1; i < N; ++i){
            int valor = int.Parse(X[i]);

            if(valor < menor_valor){
                menor_valor = valor;
                posicao = i;
            }
        }

        Console.WriteLine($"Menor valor: {menor_valor}");
        Console.WriteLine($"Posicao: {posicao}");
    }
}

File: problemas/iniciante/1180/1180.py
N = int(input())
X = [int(x) for x in input().strip().split()]

menor_valor = X[0]
posicao = 0

for i in range(1, len(X)):
    valor = X[i]

    if(valor < menor_valor):
        menor_valor = valor
        posicao = i

print(f'Menor valor: {menor_valor}')
print(f'Posicao: {posicao}')